docs/server/settings.rst

```diff
@@ -273,6 +273,14 @@ Configuration settings for applications used by Pootle.
   Enable spam prevention through a captcha.
 
 
+.. setting:: POOTLE_REPORTS_MARK_FUNC
+
+``POOTLE_REPORTS_MARK_FUNC``
+  Default: ''
+
+  Provide mark data for user daily activity charts in admin Pootle reports
```

Comment:
Try to forget everything you know about the existence of this function, then try to answer:
* what's this setting for?
* what can I do with it and how?